### PR TITLE
pprof: move pprof listener to separate port

### PR DESCRIPTION
**Pull request by henrikhodne**
The pprof handlers don't work well with negroni and middleware, so moving it to a raw http Server should work better.

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -126,19 +129,12 @@ func (srv *server) setupRoutes() {
...
-		srv.r.HandleFunc(`/debug/pprof/profile`, pprof.Profile).Name("pprof-profile")
-		srv.r.HandleFunc(`/debug/pprof/symbol`, pprof.Symbol).Name("pprof-symbol")
-		srv.r.HandleFunc(`/debug/pprof/trace`, pprof.Trace).Name("pprof-trace")
-	}
 }
 
 func (srv *server) setupMiddleware() {
 	srv.n.Use(negroni.NewRecovery())
 	srv.n.Use(negronilogrus.NewCustomMiddleware(srv.log.Level, srv.log.Formatter, "web"))
+
```
Comment (henrikhodne) [inline server/server.go]: oops this shouldn't be here

File: server/server.go [hunk 2]
```diff
@@ -148,6 +143,22 @@ func (srv *server) setupMiddleware() {
 	srv.n.UseHandler(srv.r)
 }
 
+func (srv *server) setupPprof() {
+	pprofMux := http.NewServeMux()
+	pprofMux.HandleFunc(`/debug/pprof/`, pprof.Index)
+	pprofMux.HandleFunc(`/debug/pprof/cmdline`, pprof.Cmdline)
+	pprofMux.HandleFunc(`/debug/pprof/profile`, pprof.Profile)
+	pprofMux.HandleFunc(`/debug/pprof/symbol`, pprof.Symbol)
+	pprofMux.HandleFunc(`/debug/pprof/trace`, pprof.Trace)
```
Comment (henrikhodne) [inline server/server.go]: These come from https://github.com/golang/go/blob/3825656e285155d40f286ff9e1e5deb60cf99094/src/net/http/pprof/pprof.go#L67-L71 fwiw

Comment (igorwwwwwwwwwwwwwwwwwwww) [inline server/server.go]: maybe add the link to the code as a comment?

**Comment by henrikhodne** [comment]
Tested in staging, works fine

**Pull request merged**